doc/gov/README.rst

```diff
@@ -1,5 +1,6 @@
-Cloud iNfrastructure Telco Taskforce - Community Guidelines
-===========================================================
+====================
+Community Guidelines
```

Comment:
Should we make it specific as in:

Anuket Specifications Development 

or 

Community Guidelines for Anuket Specifications Development

Reply:
Community Guidelines for Anuket Specifications Development is what I would recommend

Reply:
Fixed in [21a72fe](https://github.com/cntt-n/CNTT/pull/3119/commits/21a72fecd7dd3cd18ee030aae58c8d56d8b4f957)